neanderthal/src/main/kotlin/au/com/outware/neanderthal/presentation/presenter/VariantListPresenter.kt

```diff
@@ -69,16 +77,47 @@ class VariantListPresenter @Inject constructor(): Presenter {
     }
 
     fun onEditClicked() {
-        view.goToEditVariant(currentVariantName)
+        if(variants.size > 0) {
+            view.goToEditVariant(currentVariantName)
+        }
     }
 
     fun onDeleteClicked() {
```

Comment:
when a variant is deleted, you should notify menu options changed in the activity, and then in the activity override onPrepareOptionsMenu and get it to hide the edit & delete buttons if there are no variants. that would make these variants.size checks redundant